AndelaOSP/Andela-Socials
# #165718179 Cleanup frontend for MVP release

#### What Does This PR Do?
- Cleans up the client codebase to remove components that are not required in the MVP

#### Description Of Task To Be Completed
- Remove components that are not part of the MVP
- Adjust NavMenu styling

#### Any Background Context You Want To Provide?

#### How can this be manually tested?
- Pull this branch
- Start the app and navigate to `localhost:9000`
- Non-MVP UI components should not be visible

#### What are the relevant pivotal tracker stories?
[#165718179](https://www.pivotaltracker.com/story/show/165718179)

#### Screenshot(s)
